cmd/notary/main.go

```diff
@@ -98,7 +99,7 @@ func (n *notaryCommander) parseConfig() (*viper.Viper, error) {
 	// Setup the configuration details into viper
 	config.SetDefault("trust_dir", defaultTrustDir)
 	config.SetDefault("remote_server", map[string]string{"url": defaultServerURL})
-
+	config.SetDefault("useNative", dontusenative)
```

Comment:
in go, `false` is the default value for booleans, so we don't actually need to set a default here.
If a user does not specify the `useNative` key in a provided config, it will automatically be false.